lib/taste_tester/tunnel.rb

```diff
@@ -44,15 +44,13 @@ def run
     end
 
     def cmd
-      if TasteTester::Config.user != 'root'
-        pid = '$$'
-      else
-        pid = '\\$\\$'
```

Comment:
It requires root to write to /etc/chef, because chef runs as root... cause it starts system services and such? So if a non-root user can change where chef gets it's config than a non-root user can get anything... tt clearly needs root access on the other side.